cfme/web_ui/menu.py

```diff
@@ -298,7 +298,7 @@ def sections(self):
                         ('containers_pods', 'Pods'),
                         ('containers_routes', 'Routes'),
                         ('containers_replicators', 'Replicators'),
-                        ('containers_services', 'Services'),
+                        ('containers_services', 'Container Services'),
```

Comment:
Aaaaah, right! This is `web_ui.menu` - then the fix above is correct. 5.5 already has `Container Services` so this is for sure is a 5.6 fix and not an upstream-only one. I don't believe cm-qe is testing upstream...